biocrnpyler/polymer.py

```diff
@@ -185,14 +185,19 @@ def reverse(self):
             part.direction = self.direction_invert(part.direction)
         self.changed()
 
-
+class NamedPolymer(OrderedPolymer):
```

Comment:
Is there any reason we can't make all Polymers named polymers? But just set the name = None for most of them?

Reply:
not sure.. i didn't want to break anything that relies on equality of orderedpolymers

Reply:
fair enough.